test/specs/index.js

```diff
@@ -657,7 +658,7 @@ test('skips lifecycle scripts with ignoreScripts is set', t => {
   })
 })
 
-test('adds install script when binding.gyp is present', t => {
+only('adds install script when binding.gyp is present', t => {
```

Comment:
I would imagine this `only()` needs to be a `test()` again before this PR can be merged...